target_chains/ethereum/contracts/contracts/entropy/Entropy.sol

```diff
@@ -480,6 +522,18 @@ abstract contract Entropy is IEntropy, EntropyState {
         return _state.providers[provider].feeInWei + _state.pythFeeInWei;
     }
 
+    function getFeeAndOriginalCommitment(
+        address provider
+    ) public view override returns (uint128 feeAmount, bytes32 commitment) {
```

Comment:
nit: this method seems rather specific to this use case -- maybe not worth a method?

Reply:
I didn't want the frontend to make 2 separate RPC calls 😅